flang/runtime/stop.cpp

```diff
@@ -152,11 +153,29 @@ void RTNAME(PauseStatementText)(const char *code, std::size_t length) {
   std::exit(status);
 }
 
+static void PrintBacktrace() {
+  // TODO: Need to parse DWARF information to print function line numbers
+  const int MAX_CALL_STACK = 999;
```

Comment:
In the runtime we use braced initialization. I also changed this to `constexpr`.
```suggestion
  constexpr int MAX_CALL_STACK{999};
```